examples_tests/22.RaytracedAO/warpCommon.h

```diff
@@ -0,0 +1,25 @@
+#ifndef _WARP_COMMON_H_INCLUDED_
+#define _WARP_COMMON_H_INCLUDED_
+
+#include "common.h"
+
+#define LUMA_MIP_MAP_GEN_WORKGROUP_DIM 16
+#define WARP_MAP_GEN_WORKGROUP_DIM 16
+
+#define MAX_LUMINANCE_LEVELS 13
```

Comment:
shouldn't these be generated from and injected by Renderer.cpp at runtime?

Reply:
I will do that since it's going to be an extension, but it's shared via header inclusion between both glsl and cpp